docs/release-notes/2.15.0.asciidoc

```diff
@@ -0,0 +1,63 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.15.0]]
+== {n} version 2.15.0
+
+
+
+
+[[enhancement-2.15.0]]
+[float]
+=== Enhancements
+
+* Clarify high availability recommendations in Elasticsearch orchestration docs {pull}8151[#8151]
+* Log when k8s resources are created/updated/deleted successfully {pull}8094[#8094]
+* More meaningful error in readiness script {pull}8091[#8091] (issue: {issue}8088[#8088])
+* Update Maps Images to multi-arch {pull}8085[#8085] (issue: {issue}8034[#8034])
+* Add missing metricsets {pull}8048[#8048] (issue: {issue}7277[#7277])
```

Comment:
Let's maybe clarify what metricset, and that this is in the context of stack monitoring?